src/main/java/net/cytonic/cytosis/Cytosis.java

```diff
@@ -148,10 +151,19 @@ public static void completeNonEssentialTasks(long start) {
         COMMAND_HANDLER.setupConsole();
         COMMAND_HANDLER.registerCystosisCommands();
 
+        Logger.info("Initializing database");
+        database = new Database();
+        database.connect();
+        database.createChatTable();
+
         // Start the server
         Logger.info("Server started on port 25565");
         MINECRAFT_SERVER.start("0.0.0.0", 25565);
+        MinecraftServer.getSchedulerManager().buildShutdownTask(() -> {
+            database.disconnect();
+            Logger.info("Good night!");
+        });
         long end = System.currentTimeMillis();
-        Logger.info(StringTemplate.STR."Server started in \{end - start}ms!");
+        Logger.info(STR."Server started in \{end - start}ms!");
```

Comment:
This isn't the place for a database for a multitude of reasons. First, any call to the database needs to be on a worker thread. Second, this is really low level control that should be placed into a manager class.